### PR TITLE
Update The Silver Steed and The Many-Faced God to their errata´d versions

### PR DESCRIPTION
**Pull request by EvadLhorg**

### CODE REVIEW

**Review by Patane97 — APPROVED** [review]
Looks good, although we will need to update the `imageUrl` of The Silver Steed once we get the images uploaded. That is a low priority task though, so we can do it in a separate Pull Request after the images are actually uploaded. :)

File: packs/HMW.json [hunk 1]
```diff
@@ -1046,7 +1046,23 @@
             "text": "Your plot deck cannot include <i>Kingdom</i> plot cards.\nReduce the cost of the first card you ambush each round by 1.\n<b>Action:</b> Kneel your faction card to choose a character and a player. Until the end of the phase, that character gains each of the printed challenge icons, keywords, faction affiliations, and <i>Traits</i> of the top character in that player's dead pile.",
```
Comment (stopfstedt) [inline packs/HMW.json:1046]: needs to be updated with the v2.0 text.

File: packs/TKP.json [hunk 2]
```diff
@@ -295,7 +295,23 @@
             ],
             "text": "<i>Dothraki</i> character or Daenerys Targaryen only.\nWhile attached character is participating in a [power] challenge, it gains renown.\n<b>Reaction:</b> After attached character gains power using renown, sacrifice The Silver Steed. You may initiate an additional [power] challenge this phase.",
```
Comment (stopfstedt) [inline packs/TKP.json:296]: needs to be updated with the v2.0 text.

**Review by stopfstedt — CHANGES_REQUESTED** [review]
the errata sections look good, but the cards' primary texts needs to be realigned with the changes from their last erratum.

**Review by stopfstedt — APPROVED** [review]
pushing this over the finishing line.

**Pull request merged**